### PR TITLE
feat: Add retry logic to create ngrok tunnel

### PR DESCRIPTION
**Pull request by marinhojess**
- Add retry logic to create ngrok tunnel
- Will try to create the ngrok tunnel 3 more times if the first attempt fails

Signed-off-by: Jessica Marinho <jessica@kubeshop.io>

### CODE REVIEW

**Review by 6za — APPROVED** [review]
Loved the function pointer.

File: pkg/ngrok.go [hunk 1]
```diff
@@ -41,6 +53,33 @@ func RunNgrok(ctx context.Context) {
 	}
 }
 
+func init() {
```
Comment (jarededwards) [inline pkg/ngrok.go:56]:
```suggestion
func init() {
```
is this function being used? i didnt see a reference in your code?

Comment (jarededwards) [reply pkg/ngrok.go:56]: how often is this failing? with all the hard work of cleaning up logs and updating the logger to produce better output is this going to produce 10 times of unnecessary noise?

Comment (marinhojess) [reply pkg/ngrok.go]: @jarededwards you are right, forgot to remove, done! thanks!

**Review by 6za — APPROVED** [review]

**Review by pagottoo — APPROVED** [review]
lgtm

**Pull request merged**